modules/oe_content_news/oe_content_news.post_update.php

```diff
@@ -23,3 +25,36 @@ function oe_content_news_post_update_00001_update_field_labels(array &$sandbox):
     $field_config->save();
   }
 }
+
+/**
+ * Add type field.
+ */
+function oe_content_news_post_update_00002(): void {
+  $storage = new FileStorage(drupal_get_path('module', 'oe_content_news') . '/config/post_updates/00002');
+  $config_manager = \Drupal::service('config.manager');
+  $entity_manager = \Drupal::entityTypeManager();
+
+  // Create new field.
+  $config_ids = [
+    'field.storage.node.oe_news_types',
+    'field.field.node.oe_news.oe_news_types',
+  ];
+  foreach ($config_ids as $config_id) {
+    $config_record = $storage->read($config_id);
+    $entity_type = $config_manager->getEntityTypeIdByName($config_id);
+    $entity_storage = $entity_manager->getStorage($entity_type);
+    $entity = $entity_storage->createFromStorageRecord($config_record);
+    $entity->save();
+  }
+
+  // Update form display.
+  /** @var \Drupal\Core\Entity\Display\EntityFormDisplayInterface $form_display */
+  $values = $storage->read('core.entity_form_display.node.oe_news.default');
+  $display = EntityFormDisplay::load($values['id']);
+  if ($display) {
+    foreach ($values as $key => $value) {
+      $display->set($key, $value);
+    }
+    $display->save();
+  }
+}
```

Comment:
Split this into 2 update path:
1. Create new field.
2. Update form display.

so that if one fail on prod we know which one.